src/index.js

```diff
@@ -6,24 +6,78 @@ import './css/base.scss';
 
 //JS FILES
 import Traveler from './traveler.js';
+import Trip from './trip.js';
 import fetchAPI from './fetch.js';
+import domUpdates from './domUpdates.js';
 
 //QUERY SELECTORS
+const bookAFlightBtn = document.querySelector('.book-now-btn');
+const claculateNewTripCostBtn = document.querySelector('.calculate-cost-button');
+const returnToDashboardBtn = document.querySelector('.return-to-dashboard');
+const submitTripBtn = document.querySelector('.submit-request');
+const formDestinationInput = document.querySelector('.destinations-list');
+const formDurationInput = document.querySelector('.select-duration');
+const formTravelersInput = document.querySelector('.select-travelers');
+const formDateInput = document.querySelector('.select-date');
 
 //EVENT LISTENERS
 window.addEventListener('load', generateUsersInfo);
+bookAFlightBtn.addEventListener('click', function() {
+  domUpdates.displayBookFlightForm(destinationsData);
+});
+claculateNewTripCostBtn.addEventListener('click', claculateNewTripCost);
+returnToDashboardBtn.addEventListener('click', domUpdates.returnToDashboard);
+submitTripBtn.addEventListener('click', addTripRequest);
 
 //GLOBAL VARIABLES
 let currentUser;
 let destinationsData;
+let tripsData;
 
-//FETCH DATA
+//FETCH DAT
```

Comment:
Fetch Dat, all on the floor. Fetch Dat, give me some more. Fetch Dat, till you get sore. Fetch Dat. Waaaooooaaooahh